resources/release_note/js/appRN.js

```diff
@@ -81,7 +81,12 @@
     }
     
     function createUrl(host){
-        return "http://"+ host + "." + window.location.host+ (window.location.host.indexOf(".xip.io") > -1 ? "" : ".xip.io/");
+        var ipformat = /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/;
+        if (window.location.hostname.match(ipformat)) {
+            return "http://" + host + "." + window.location.host + ".xip.io";
+        } else {
+            return "http://" + host + "." + window.location.host;
```

Comment:
same on this line: 

by using the ` window.location.protocol ` method?

it should look something like this:
`return window.location.protocol + "//" + host + "." + window.location.host;`